docs/Configuration.md

```diff
@@ -6,57 +6,55 @@ nav_order: 4
+| spark.gluten.sql.native.bloomFilter                     | Enable of Disable native runtime bloom filter.                                                                                                                                                                                                                                                                                 | true                                                 |
+| spark.gluten.sql.columnar.wholeStage.fallback.threshold | Configure the threshold for whether whole stage will fall back in AQE supported case by counting the number of ColumnarToRow & vanilla leaf node                                                                                                                                                                               | \>= 3                                                |
+| spark.gluten.sql.columnar.maxBatchSize                  | Set the number of rows for the output batch                                                                                                                                                                                                                                                                                    | 4096                                                 |
+| spark.gluten.loadLibFromJar                             | Gluten will load dynamic link library from jars for gluten/cpp.                                                                                                                                                                                                                                                                | false                                                |
+| spark.gluten.sql.columnar.force.hashagg                 | Force to use hash agg to replace sort agg.                                                                                                                                                                                                                                                                                     | true                                                 |
+| spark.gluten.sql.columnar.vanillaReaders                | Enable vanilla spark's vectorized reader. Please note it may bring perf. overhead due to extra data transition. We recommend to disable it if most queries can be fully offloaded to gluten.                                                                                                                                   | false                                                |
```

Comment:
Please remove this line. Since this config property (`spark.gluten.sql.columnar.vanillaReaders`) is not supported in 1.0 release. Thanks!